README.md

```diff
@@ -0,0 +1,114 @@
+# xtm - a library for cross-thread messaging with event loop intergation
+
+The library provides the following facilities:
+
+# xtm_queue
+
+Opaque struct, that represent one-directional, one-reader-one-writer queue
+implementation with eventloop integration ability.
+
+## xtm_create
```

Comment:
If it is supposed to allocate the queue, it must be `xtm_queue_new()`. `queue`, because all methods of a struct should have its name as a prefix. So all methods must begin with `xtm_queue_*`. `new` because this is our code style agreement to use `new`/`delete` for allocation/deletion, and `create`/`destroy` for creation and destruction in-place.
The same prefix-comment about all the other functions.

Reply:
I talked with @Mons about it. He agreed to rename xtm_create/xtm_delete to xtm_queue_new/xtm_queue_delete, but he asked not to touch the names of other functions